http/http_exception.cc

```diff
@@ -24,6 +24,14 @@ void rethrowHttpException(int httpCode, const Utf8String & message, Any details)
         details2["context"]["details"] = jsonEncode(http.details);
         details2["context"]["error"] = http.message;
         throw HttpReturnException(httpCode == -1 ? http.httpCode : httpCode, message, details2);
+    } catch (const std::bad_alloc & exc) {
```

Comment:
I seem to recall a discussion about avoiding allocations when handing std::bad_alloc. I doubt such a small amount of memory would be a problem but I think it's better to think twice about this.